dev/docker-compose.yml

```diff
@@ -43,6 +43,7 @@ services:
       # TODO: Where should we be running rspec tests from, ideally?
       # See https://github.com/DatabaseCleaner/database_cleaner#safeguards
       DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL: "true"
+      CONJUR_ALLOW_USER_CREATION: false
```

Comment:
As this is primarily for SaaS, let's default this to `true` if at all.

Reply:
OK - I will remove it from here at all in order to keep the existing conjur-oss functionality